test/cross-browser-testing/browserstack/browserstack.karma.config.js

```diff
@@ -0,0 +1,73 @@
+const { DEBUG } = process.env;
+
+let files = [
+    '../../lib/mockhttprequest.js',
+    '../../lib/geomock.js',
+    '../../../node_modules/fetch-mock/dist/es5/client-bundle.js',
+    '../../../node_modules/sinon-browser-only/sinon.js',
+    '../../config.js',
+    '../../../dist/mparticle.js',
+    '../../test-bundle.js'
+];
+
+let singleRun = true;
```

Comment:
Same here. This isn't changing is it?

```suggestion
const singleRun = true;
```